libraries/freertos_plus/standard/pkcs11/include/iot_pkcs11.h

```diff
@@ -1,6 +1,6 @@
 /*
- * Amazon FreeRTOS
- * Copyright (C) 2017 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
+ * Amazon FreeRTOS PKCS#11 V1.0.0
```

Comment:
This is supposed to be PKCS #11 (space in between, this was a comment I got in 2017).

Reply:
Also I think it is at V2.0.0?

Reply:
Yea. I followed the same pattern in the 

libraries/abstractions/pkcs11 folder for versioning. This was done by Alexa. Do you think both those needs modification?

Reply:
yea those are all wrong lol